test/unit/processing/components/get-remaining-amount.test.js

```diff
@@ -0,0 +1,19 @@
+const getRemainingAmount = require('../../../../app/processing/components/get-remaining-amount')
```

Comment:
This is really difficult to know if this test case is complete given the lack of ACs attached to the ticket.

Anwar may have something to say on this.

For now, seems fine to me.